reference/configuration/framework.rst

```diff
@@ -1443,19 +1443,24 @@ errors.
 session
 ~~~~~~~
 
-storage_id
-..........
+storage_factory_id
```

Comment:
```suggestion

.. storage_id:

storage_factory_id
```

to keep the anchor working

Reply:
Added `.. _storage_id:` after merging. Thanks!